packages/visx-tooltip/test/useTooltipInPortal.test.tsx

```diff
@@ -1,7 +1,46 @@
-import { useTooltipInPortal } from '../src';
+import React from "react";
+import { shallow } from "enzyme";
+import ResizeObserver from "resize-observer-polyfill";
+import { useTooltipInPortal } from "../src";
+import { UseTooltipPortalOptions } from "../src/hooks/useTooltipInPortal";
 
-describe('useTooltipInPortal()', () => {
-  test('it should be defined', () => {
+interface TooltipWithZIndexProps {
+  zIndexOption?: UseTooltipPortalOptions["zIndex"];
+  zIndexProp?: UseTooltipPortalOptions["zIndex"];
+}
+
+const TooltipWithZIndex = ({
+  zIndexOption,
+  zIndexProp,
+}: TooltipWithZIndexProps) => {
+  const { TooltipInPortal } = useTooltipInPortal({
+    polyfill: ResizeObserver,
+    zIndex: zIndexOption,
+  });
+  return <TooltipInPortal zIndex={zIndexProp}>Hello</TooltipInPortal>;
+};
+
+describe("useTooltipInPortal()", () => {
+  test("it should be defined", () => {
     expect(useTooltipInPortal).toBeDefined();
   });
+
+  it("should pass zIndex prop from options to Portal", () => {
+    const wrapper = shallow(<TooltipWithZIndex zIndexOption={1} />, {
+      disableLifecycleMethods: true,
+    }).dive();
+    const zIndex = wrapper.find("Portal").prop("zIndex");
+    expect(zIndex).toEqual(1);
+  });
+
+  it("should pass zIndex prop from component to Portal", () => {
+    const wrapper = shallow(
+      <TooltipWithZIndex zIndexOption={1} zIndexProp="var(--tooltip-zindex)" />,
```

Comment:
@kangaechigai , is the setting for `zIndexOption` needed here since you're setting the `zIndexProp`?

Reply:
Not _needed_, but I wanted to illustrate that the prop overrides the option.